### ISSUE TITLE
SERVER_NAME missing

### ISSUE THREAD

**Issue by revmischa** [slank/awsgi#13]
```
  File "/var/task/vendor/awsgi/__init__.py", line 21, in response
    output = app(environ(event, context), sr)
  File "/var/task/vendor/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/var/task/vendor/flask/app.py", line 1977, in wsgi_app
    ctx = self.request_context(environ)
  File "/var/task/vendor/flask/app.py", line 1938, in request_context
    return RequestContext(self, environ)
  File "/var/task/vendor/flask/ctx.py", line 242, in __init__
    self.url_adapter = app.create_url_adapter(self.request)
  File "/var/task/vendor/flask/app.py", line 1765, in create_url_adapter
    server_name=self.config['SERVER_NAME'])
  File "/var/task/vendor/werkzeug/routing.py", line 1307, in bind_to_environ
    wsgi_server_name = environ['SERVER_NAME']
KeyError: 'SERVER_NAME'
```

Using `sam local start-api`

**Comment by slank** [comment]
Hi, @revmischa. Thanks for reporting this issue. I'm guessing that the client you're using doesn't set a `Host` header. We should probably handle that by setting `SERVER_NAME` to `""` since `SERVER_NAME` is required by the [wsgi spec](https://www.python.org/dev/peps/pep-0333/). Same for `SERVER_PORT`.

I'll try to get a PR made today.

**Comment by revmischa** [comment]
Yeah I'm just testing with https://github.com/awslabs/aws-sam-local

**Comment by frob** [comment]
Did anything ever happen here? Where could I make this change?

**Comment by slank** [comment]
Fixed by #20

**Comment by meetkakadiya44** [comment]
> Yeah I'm just testing with https://github.com/awslabs/aws-sam-local

did you get to solve that issue
